platform/runner-plugin/std/sys/src/main/kotlin/io/hamal/plugin/std/sys/Plugin.kt

```diff
@@ -66,6 +66,7 @@ class PluginSysFactory(
 
                 "func_create" to FuncCreateFunction(sdk),
                 "func_deploy" to FuncDeployFunction(sdk),
+                "func_deployments" to FuncDeploymentsFunction(sdk),
```

Comment:
```suggestion
                "func_deployment_list" to FuncDeploymentListFunction(sdk),
```

to be consistent with the rest